nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/test/java/org/apache/nifi/processors/elasticsearch/TestQueryElasticsearchHttpNoHits.java

```diff
@@ -142,7 +142,7 @@ public void testQueryElasticsearchOnTrigger_Hits_NoHits() throws IOException {
                 runner.assertValid();
 
                 runner.setIncomingConnection(false);
-                runAndVerify(3,1,0,true);
+                runAndVerify(3,0,0,true);
```

Comment:
Should we run this with both 1 and 0 for coverage? I haven't looked into the parameter, just shooting from the hip :)

Reply:
It's been a long one, what scenario are you thinking about?

Reply:
Nevermind, I didn't know what that parameter was for, zero is the correct value